data/ecosystem/vendors.yaml

```diff
@@ -384,3 +384,10 @@
   contact: 'https://github.com/adnanrahic'
   oss: true
   commercial: true
+- name: Alibaba Cloud
+  distribution: true
```

Comment:
can you provide evidence for this distribution? Documentation or GitHub repository link

Reply:
> if your documentation is not released yet then I suggest you set distribution to "no" and come back with an update when the documentation is available.

ok, I adjusted it.